.clangd

```diff
@@ -0,0 +1,7 @@
+CompileFlags:
```

Comment:
I added this for linux dev's, is that OK?

Reply:
This breaks it for me since my .clangd looks different...